Oblig 2 Ferdig - Sondre

RevanInHeaven opened the pull request:
Sondre Alexander Mo Didriksen

RevanInHeaven committed:
Add files via upload

Bsingstad committed:
Lagt til kommentarer i rødt i notebookene

Bsingstad commented:
Veldig godt arbeid i Oppgave 1, men i oppgave 2 gjør du noen elementere feil som ødelegger for deg. Jeg må derfor underkjenne obligen totalt sett, men du vil få en mulighet til å rette opp i feilene innen 1 Desember

RevanInHeaven committed:
Add files via upload

RevanInHeaven committed:
Add files via upload

RevanInHeaven committed:
Tidligere requirements.txt ville ikke erstattes automatisk med den nye

Prøvde å sette inn den nye requirements.txt filen, men den ville ikke erstatte den som er her så jeg prøver å slette den gamle å laste inn den nye.

RevanInHeaven committed:
Den nye requirements.txt filen.

RevanInHeaven commented:
Har fikset flere av feilene i Oppgave 2 filen. Oppgave 1 filen har ikke blitt forandret. requirements.txt filen er nå oppdatert for den nye Oppgave 2 filen. Filene er klare for å bli vurdert igjen.

Bsingstad committed:
Update Oppgave 2.ipynb

Bsingstad commented:
Veldig bra jobba! Dette var noe helt annet. Har lagt igjen noen kommentarer, men du behøver ikke svare på de, jeg flytter deg uansett over på godkjent
